rules/cross-platform/indicator-match.toml

```diff
@@ -0,0 +1,148 @@
+author = ["Elastic"]
+created_at = "2021-03-17T16:07:18.075Z"
+updated_at = "2021-03-25T20:30:59.721Z"
```

Comment:
```suggestion
created_at = "2021/03/17"
updated_at = "2021/03/25"
```

Reply:
also, this structure looks incorrect.
can you update it to look like other rules? `[metadata]` + `[rule]`